fictadvisor-web/src/components/pages/schedule-page/schedule-event-edit-section/schedule-info-card/ScheduleInfoCard.tsx

```diff
@@ -111,25 +111,31 @@ const ScheduleInfoCard: FC<ScheduleInfoCardProps> = ({
             color={TagColorMapper[event.eventType]}
           />
         )}
-        <Typography variant="body1Medium">Викладач</Typography>
-        <Box sx={styles.teachersContainer}>
-          {loading ? (
-            <Skeleton {...skeletonProps} width={200} height={45} />
-          ) : !event!.teachers || event!.teachers.length === 0 ? (
-            <Typography variant="body1Medium">
-              Інформація про викладачів відсутня
-            </Typography>
-          ) : (
-            event?.teachers.map(teacher => (
-              <Link
-                key={teacher.id}
-                href={`/teachers/${teacher.id}`}
-                text={`${teacher.firstName} ${teacher.middleName} ${teacher.lastName}`}
-                target={'_blank'}
-              />
-            ))
-          )}
-        </Box>
+        {event?.eventType !== 'OTHER' ? (
```

Comment:
put this in some descriptive variable like `isDisciplineRelatedEvent`,

otherwise, good job troubleshooting this giant pile of shi...